contributors/join-us/staff-recruiting.md

```diff
@@ -217,7 +210,7 @@ This process will be managed by HR.
 
 1. Once an offer is approved, the Recruiter will select the appropriate offer template based on candidates’s residence country/region and send for signatures.
 2. Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever and export the profile to BambooHR.
```

Comment:
Upon full execution of a new staff contract, the Recruiter will mark the candidate as **Hired** in Lever.